ts/screens/ingress/CheckBox.tsx

```diff
@@ -17,25 +16,24 @@ const styles = StyleSheet.create({
     height: 20,
     alignItems: "center",
     justifyContent: "center",
+    borderWidth: 2,
     borderColor: checkBoxColor,
-    borderWidth: 2
+    borderRadius: 4
+  },
+  checked: {
+    borderColor: "transparent",
+    backgroundColor: checkBoxColor
   }
 });
 
-export const IngressCheckBox = (props: Props) => {
-  const checkboxBackgroundColor = props.checked
-    ? checkBoxColor
-    : transparentColor;
-
-  return (
-    <View style={[styles.base, { backgroundColor: checkboxBackgroundColor }]}>
-      {props.checked && (
-        <IconFont
-          name={checkBoxIcon}
-          size={styles.base.width * 0.8}
-          color={IOColors.white}
-        />
-      )}
-    </View>
-  );
-};
+export const IngressCheckBox = (props: Props) => (
+  <View style={[styles.base, props.checked && styles.checked]}>
```

Comment:
When `checked` is false it would produce `[styles.base, false]` as a result, that might work in practice. However I would prefer being more explicit like this: 
```suggestion
  <View style={[styles.base, props.checked ? styles.checked : {}]}>
```

What do you think?